### ISSUE TITLE
Missing import for asyncio in README example code

### ISSUE THREAD

**Issue by okamiRvS** [openai/openai-python#1825]
### Confirm this is an issue with the Python library and not an underlying OpenAI API

- [X] This is an issue with the Python library

### Describe the bug

Hi there,

I noticed a small typo in the example code provided in the README file. The code snippet uses *asyncio.run(main())* but is missing the import statement for *asyncio*. This could cause confusion for users running the example as it will result in a *NameError: name 'asyncio' is not defined*.

### To Reproduce

Here's the updated code with the necessary import:

```python
from openai import AsyncOpenAI
import asyncio  # Add this import

client = AsyncOpenAI()

async def main():
    stream = await client.chat.completions.create(
        model="gpt-4",
        messages=[{"role": "user", "content": "Say this is a test"}],
        stream=True,
    )
    async for chunk in stream:
        print(chunk.choices[0].delta.content or "", end="")

asyncio.run(main())
```

### OS

-

### Python version

-

### Library version

-

**Comment by baslia** [comment]
Hey, I think this is solved now.

**Comment by okamiRvS** [comment]
I still see the absence of the asynco import in the second snippet of "streaming resppnse"

**Comment by baslia** [comment]
My bad, I just made a PR for that: https://github.com/openai/openai-python/pull/1858

**Comment by RobertCraigie** [comment]
thanks @baslia, this was fixed in #1858
